datacube_stats/utils/create_image_demo.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
```

Comment:
This file probably shouldn't live here, and contains lots of copy-pasted code from elsewhere.